rpc/js/src/dial.ts

```diff
@@ -29,57 +29,75 @@ import { SignalingService } from './gen/proto/rpc/webrtc/v1/signaling_pb_service
 import { addSdpFields, newPeerConnectionForClient } from './peer';
 
 export interface DialOptions {
+  // TODO: Add doc comments for these properties
   authEntity?: string | undefined;
   credentials?: Credentials | undefined;
   webrtcOptions?: DialWebRTCOptions;
   externalAuthAddress?: string | undefined;
   externalAuthToEntity?: string | undefined;
 
-  // `accessToken` allows a pre-authenticated client to dial with
-  // an authorization header. Direct dial will have the access token
-  // appended to the "Authorization: Bearer" header. WebRTC dial will
-  // appened it to the signaling server communication
-  //
-  // If enabled, other auth options have no affect. Eg. authEntity, credentials,
-  // externalAuthAddress, externalAuthToEntity, webrtcOptions.signalingAccessToken
+  /**
+   * `accessToken` allows a pre-authenticated client to dial with
+   * an authorization header. Direct dial will have the access token
+   * appended to the "Authorization: Bearer" header. WebRTC dial will
+   * appened it to the signaling server communication
+   *
+   * If enabled, other auth options have no affect. Eg. authEntity,
+   * credentials, externalAuthAddress, externalAuthToEntity,
+   * webrtcOptions.signalingAccessToken
+   */
   accessToken?: string | undefined;
 }
 
 export interface DialWebRTCOptions {
+  // TODO: Add doc comments for these properties
```

Comment:
```suggestion
  // `disableTrickleICE` forces ICE agents to wait until all ICE candidates are gathered before performing connectivity checks.
```

some references:
* https://datatracker.ietf.org/doc/html/draft-ietf-mmusic-trickle-ice
* https://webrtchacks.com/trickle-ice/